src/js/oTable.js

```diff
@@ -131,219 +23,347 @@ function descendingSort(...args) {
 	return 0 - ascendingSort.apply(this, args);
 }
 
-/**
- * Sorts the table by a specific column
- * @param  {number} The index of the column to sort the table by
- * @param  {bool} Which direction to sort in, ascending or descending
- * @param  {bool} Whether the values in this column are numeric, if they are numeric we convert the contents into numbers
- * @returns undefined
- */
-OTable.prototype.sortRowsByColumn = function (index, sortAscending, isNumericValue) {
-	const tbody = this.rootEl.querySelector('tbody');
-	const rows = Array.from(tbody.querySelectorAll('tr'));
-	const intlCollator = getIntlCollator();
-	rows.sort(function (a, b) {
-		let aCol = a.children[index];
-		let bCol = b.children[index];
-
-		if (aCol.getAttribute('data-o-table-order') !== null) {
-			aCol = aCol.getAttribute('data-o-table-order');
-			bCol = bCol.getAttribute('data-o-table-order');
-			if (!isNaN(parseInt(aCol, 10))) {
-				aCol = parseInt(aCol, 10);
-				bCol = parseInt(bCol, 10);
-			}
+function wrapElement(targetEl, wrapEl) {
+	const parentEl = targetEl.parentNode;
+	parentEl.insertBefore(wrapEl, targetEl);
+	wrapEl.appendChild(targetEl);
+}
+
+class OTable {
+	/**
+	 * Initialises an o-table components inside the element passed as the first parameter
```

Comment:
> Initialises an o-table _component_ inside the element passed as the first parameter

Pedantic: The constructor first checks to see if the element itself is an o-table component, if it is not it then looks inside the element to find the first instance of an o-table and constructs that.